nsis/src/nsis/NsisLoader.java

```diff
@@ -158,7 +156,7 @@ private void checkHeaderCompression(Program program, NsisScriptHeader header, Ta
 		System.out.print("Header is Z compressed\n");
 	}
 
-	private void processBlockHeaders(Program program, TaskMonitor monitor, BinaryReader reader) {
+	private void processBlockHeaders(Program program, TaskMonitor monitor, BinaryReader reader, long nsis_header_offset) {
```

Comment:
nit: Not for this PR, but I wonder now that the NsisExecutable exists, the logic for parsing blocks should be moved there.

Reply:
Yes I think so too! Thought it was good to do it in another PR so this one is not too big. I think all the NSIS parsing logic should be moved in NsisExecutable.java